templates/employee.json

```diff
@@ -1,365 +1,77 @@
-      "operation": "get",
-      "properties": {
-        "url": "employees"
-      },
-      "supports_since": true
-    },
+    "_id": "{{@ system @}}-{{@ datatype @}}-collect",
+    "add_namespaces": false,
+    "exclude_completeness": [
+      "{{@ system @}}-{{@ datatype @}}-share"
+    ],
     "sink": {
       "set_initial_offset": "onload"
     },
-    "add_namespaces": false
-  },
-  {
-    "_id": "{{@ system @}}-employee-share-insert",
-    "type": "pipe",
     "source": {
-      "type": "dataset",
-      "dataset": "{{@ system @}}-employee-transform"
-    },
-    "sink": {
-      "set_initial_offset": "onload"
+      "operation": "{{@ datatype @}}-list",
+      "supports_since": true,
+      "system": "{{@ system @}}",
+      "trace": {},
+      "type": "rest"
     },
     "transform": [
       {
-        "type": "dtl",
-        "rules": {
-          "default": [
-            [
-              "add",
-              "::properties",
-              [
-                "dict",
-                "url",
-                "employees"
-              ]
-            ],
-            [
-              "copy",
-              "sesam-id"
-            ],
-            [
-              "add",
-              "_old-id",
-              [
-                "first",
-                [
-                  "hops",
-                  {
-                    "datasets": [
-                      "{{@ system @}}-employee-share-insert f"
-                    ],
-                    "where": [
-                      [
-                        "eq",
-                        "_S.sesam-id",
-                        "f.sesam-id"
-                      ]
-                    ],
-                    "return": "f.id",
-                    "track-dependencies": false
-                  }
-                ]
-              ]
-            ],
-            [
-              "add",
-              "::payload",
-              [
-                "apply",
-                "payload",
-                "_S."
-              ]
-            ],
-            [
-              "discard",
-              [
-                "is-empty",
-                "_T.payload.id"
-              ]
-            ]
-          ],
-          "payload": [
-            [
-              "copy",
-              "*"
-            ],
-            [
-              "remove",
-              "_*"
-            ],
-            [
-              "remove",
-              "sesam-id"
-            ],
-            [
-              "if",
-              [
-                "is-not-empty",
-                "_P._T._old_id"
-              ],
-              [
-                "add",
-                "id",
-                "_T._old-id"
-              ]
-            ]
-          ]
-        }
-      },
-      {
-        "type": "rest",
-        "system": "{{@ system @}}",
-        "operation": "insert",
-        "replace_entity": false
+        "properties": {
+          "primary_key": "id",
+          "rest_system": "{{@ system @}}",
```

Comment:
You don't need rest_system either if you aren't doing a rest lookup in the template